src/App.jsx

```diff
@@ -1088,12 +1088,38 @@ const App = memo(() => {
     if (!playing) {
       video.play();
     } else {
+      let calculatedRate;
+      if (useMultiplier) {
+        switch (true) {
+          case speedMultiplier === '2x' && dir > 0:
+            calculatedRate = video.playbackRate * 2.0;
+            break;
+          case speedMultiplier === '2x' && dir < 0:
+            calculatedRate = video.playbackRate * 0.5;
+            break;
+          case speedMultiplier === '1.1x' && dir > 0:
+            calculatedRate = video.playbackRate * 1.1;
+            break;
+          case speedMultiplier === '1.1x' && dir < 0:
+            calculatedRate = video.playbackRate * 0.9091;
+            break;
+          default:
+            calculatedRate = 1.0;
```

Comment:
When does this happen?

Reply:
config file: `speedMultiplier: "fooBar"`, this avoids multiplying with `null` 😬

Reply:
I think in the default case it would be better to default to a multiplier value of maybe 1.1, instead of just setting playback rate to 1 all the time